### PR TITLE
UG document for Themes for WinUI SfScheduler control.

### PR DESCRIPTION
**Pull request by VidyalakshmiMani**
Added UG document for themes in the WinUI SfScheduler.

### CODE REVIEW

File: winui/Scheduler/Themes.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
```
[the remark this answers is not in the document]
Comment (VidyalakshmiMani) [reply winui/Scheduler/Themes.md:1]: Added.

File: winui/Scheduler/Themes.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+ <scheduler:SfScheduler>
+
+     <scheduler:SfScheduler.Resources>
+         
+         <ResourceDictionary>
+             <ResourceDictionary.ThemeDictionaries>                 
+                 <ResourceDictionary x:Key="Dark">
+                      <SolidColorBrush x:Key="SyncfusionMonthCellBackground" Color="Beige" />
+                      <SolidColorBrush x:Key="SyncfusionMonthCellForeground" Color="Purple" />
```
Comment (Karthickmani97) [inline winui/Scheduler/Themes.md:99]: Change different color to looks UI better.

File: winui/Scheduler/Themes.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+     </scheduler:SfScheduler.Resources>
+
+ </scheduler:SfScheduler>
+{% endhighlight %}
+{% endtabs %}
+
+![customize-themes-in-winui-scheduler](Themes_Images/customize-theme-in-winui-sfscheduler)
+
+
+N> The HighContrast theme set in the scheduler will be applied to all existing high contrast themes on the computer.
```
Comment (Karthickmani97) [inline winui/Scheduler/Themes.md:114]: The HighContrast theme settings within the scheduler will be universally applied to all pre-existing high contrast themes on the computer.

File: winui/Scheduler/Themes.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: Themes in WinUI Scheduler control | Syncfusion
+description: Learn here all about to use various themes in Syncfusion WinUI Scheduler(SfScheduler) control and more.
```
Comment (Karthickmani97) [inline winui/Scheduler/Themes.md:4]: Learn here all about to use various themes in Syncfusion WinUI **Scheduler (SfScheduler)** control and more.

File: winui/Scheduler/Themes.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+platform: winui
+control: SfScheduler
+documentation: ug
+---
+
+# Themes in WinUI Scheduler (SfScheduler)
+Theming plays a crucial role in enhancing the visual appeal and user experience of applications. The WinUI SfScheduler  allows you to customize its appearance by applying different themes.
```
[the remark this answers is not in the document]
Comment (VidyalakshmiMani) [reply winui/Scheduler/Themes.md]: Changed.

File: winui/Scheduler/Themes.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+Refer to the following link for guidance on applying themes: [Themes for Syncfusion WinUI controls](https://help.syncfusion.com/winui/common/themes)
+
+## Keys
```
[the remark this answers is not in the document]
Comment (VidyalakshmiMani) [reply winui/Scheduler/Themes.md]: Removed.

File: winui/Scheduler/Themes.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+    </tr>
+    <tr>
+        <td>SyncfusionViewHeaderControlBorderThickness</td>
+        <td>Thickness of the border for the view header of SfScheduler.</td>
+    </tr>    
+</table>
+
+You can find the keys for all themes in this [theme resource file.](https://github.com/syncfusion/winui-controls-theme-resource-files/tree/master/Syncfusion.Scheduler.WinUI)
+
+## Customize themes
```
[the remark this answers is not in the document]
Comment (VidyalakshmiMani) [reply winui/Scheduler/Themes.md]: Changed.

File: winui/Scheduler/Themes.md (hunk 3 above)
Comment (VidyalakshmiMani) [reply winui/Scheduler/Themes.md]: Changed.

File: winui/Scheduler/Themes.md (hunk 4 above)
Comment (VidyalakshmiMani) [reply winui/Scheduler/Themes.md]: Highlighted the words.

File: winui/Scheduler/Themes.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+platform: winui
+control: SfScheduler
+documentation: ug
+---
+
+# Themes in WinUI Scheduler (SfScheduler)
+ The WinUI Scheduler allows you to apply different themes to customize its appearance for a better user exoerience.
+
+Refer to the following link for guidance on applying themes: [Themes for Syncfusion WinUI controls](https://help.syncfusion.com/winui/common/themes) 
+
```
Comment (Karthickmani97) [inline winui/Scheduler/Themes.md:14]: @VidyalakshmiMani , Update changes as we discussed through teams call.

Comment (VidyalakshmiMani) [reply winui/Scheduler/Themes.md:14]: Updated the changes as discussed.

File: winui/Scheduler/Themes.md (hunk 2 above)
Comment (VidyalakshmiMani) [reply winui/Scheduler/Themes.md]: Changed.

File: winui/Scheduler/Themes.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Themes in WinUI Scheduler control | Syncfusion
```
Comment (M-A32) [inline winui/Scheduler/Themes.md:3]: title: Themes in WinUI Scheduler Control | Syncfusion

File: winui/Scheduler/Themes.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+description: Learn here all about to use various themes in Syncfusion WinUI Scheduler (SfScheduler) control and more.
```
Comment (M-A32) [inline winui/Scheduler/Themes.md:4]: description: Learn here all about how to use various themes in Syncfusion WinUI Scheduler (SfScheduler) control, and more.

File: winui/Scheduler/Themes.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+platform: winui
+control: SfScheduler
+documentation: ug
+---
+
+# Themes in WinUI Scheduler (SfScheduler)
+ The WinUI Scheduler allows you to apply different themes to customize its appearance for a better user exoerience.
```
Comment (M-A32) [inline winui/Scheduler/Themes.md:11]: The WinUI Scheduler allows you to apply different themes to customize its appearance for a better user experience.

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: winui/Scheduler/Themes.md (hunk 9 above)
Comment (VidyalakshmiMani) [reply winui/Scheduler/Themes.md]: Changed.

File: winui/Scheduler/Themes.md (hunk 10 above)
Comment (VidyalakshmiMani) [reply winui/Scheduler/Themes.md]: Changed.

File: winui/Scheduler/Themes.md (hunk 11 above)
Comment (VidyalakshmiMani) [reply winui/Scheduler/Themes.md]: Changed.

**Review by karthikraja-arumugam — APPROVED** [review]

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**